Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1331,7 +1331,8 @@ unsigned MediaPlayerPrivateGStreamerBase::videoDecodedByteCount() const
 #if ENABLE(ENCRYPTED_MEDIA)
 void MediaPlayerPrivateGStreamerBase::cdmInstanceAttached(const CDMInstance& instance)
 {
-    ASSERT(!m_cdmInstance);
+    //ASSERT(!m_cdmInstance);
```

Comment:
// FIXME: ASSERT...

Is this going to be permanent, if yes, then we can remove the ASSERT completely.